addons/project/static/src/components/project_right_side_panel/project_right_side_panel.scss

```diff
@@ -86,4 +86,11 @@ $o-rightpanel-p-tiny: $o-rightpanel-p-small*0.5;
             width: 20%;
         }
     }
+
+    .o_rightpanel_milestone{
+
+        .o_milestone_detail{
+            width: 95%;
+        }
+    }
```

Comment:
Why do we need this? It seems now compared to your other PRs. 🤔

Reply:
Hello @xavierbol ,

In previous versions, the milestone panel on the right side was styled using custom CSS. However, starting from version 16, Bootstrap was adopted for styling, and the panel now has a fixed width of 100%. This fixed width can become an issue when long milestone names are added, as it can cause the milestone row to exceed its intended boundaries. To address this, it's recommended to adjust the width from 100% to 95%.

Thank You :)

Reply:
Hello @ravb-odoo ,

I am wondering if it is better as I did in the additional commit, can you check?

Because with your actual solution the trash icon is not correctly aligned when the milestone name is too long.

Thanks in advance! 🙂 

Have a nice day! 🙂

Reply:
Hello @xavierbol ,

I concur, this solution is more effective.

Thank you:)